### DISCUSSION TITLE
Por que as minas não participam do fórum?

### DISCUSSION THREAD

**Discussion by kvnol** [frontendbr/forum#1001] [Discussão]
Há algum tempo venho percebendo que a grande maioria da galera que interage aqui é homem e queria saber, pelas mulheres do fórum (que talvez só acompanhem), o porquê disso acontecer.

Vocês, **mulheres**, ficam desconfortáveis, são tratadas mal ou algo do tipo aqui no fórum?

**Comment by miguelzinhe** [comment]
Vou dar minha opinião, eu, pessoa nascida como mulher, porém de gênero não binário.

Não é que eu não me sinta desconfortável, mas eu tenho receio de tentar ajudar alguém aqui e ser atacada ou ser deslegitimizada como profissional. Eu sinto uma pressão enorme para ser perfeita, pra ser uma profissional que entende de tudo, especialista, e eu não consigo demonstrar muito do que eu sei. Eu tenho receio de errar e ter dedos apontados na minha cara depois me chamando de péssima dev. Eu me sinto MILHÕES de vezes mais confortável nos canais privados do Training Center ou respondendo meus ex-alunos, ex-alunas e mentorados via inbox.

O grupo do FrontEndBR no Facebook é um grupo enorme com muito cara escroto que ama tirar sarro de iniciante, por exemplo. Eu tenho receio de ser uma dessas pessoas a ser zoada.

**Comment by miguelzinhe** [comment]
Esse forum me ajuda MUITA em tomada de decisões em tecnologia, em discussões que levo sempre pro meu Twitter ou mando link pra meus amigos e amigas devs. Mas eu não sou de interagir pra não parecer "ridicula".

**Comment by alinebastos** [comment]
Minha resposta é bem parecida com a da @oieusouamiguel.
Nunca me senti confortável em foruns. 
Hoje em dia eu prefiro não participar a correr o risco de me sentir mal com alguma resposta ou nenhuma resposta (sim, pq homens respondem pra homens).
Eu tb me sinto milhões de vezes mais confortável no TC ou no meu twitter, que são ambientes que eu sei que eu tenho voz, que eu posso responder a altura, que eu posso cobrar, que eu não vou me sentir intimidada.
É bem complicado mudar isso de uma hora pra outra.

**Comment by alinebastos** [comment]
Por exemplo, aqui tem moderadoras?
Um dos motivos que as mulheres tem participado MUITO do TC é pq tem moderadoras bem ativas. Elas sabem que sempre tem gente de olho, aí se sentem mais seguras.

**Comment by yoneshinzato** [comment]
eu também! pelos mesmos motivos da Miguel! Medo de ser rechaçada, de falar uma besteira e tirarem sarro, de sofrer algum tipo de reprimenda. Pior ainda pra mim que sei bem pouco , sou novata e quero entrar na área. O medo de "represálias" cala muitas mulheres....

**Comment by brunover** [comment]
Boa Miguel, Aline e Yone, obrigado por compartilhar os sentimentos.

Se possível falem mais do que sentem falta por aqui e que acham que
melhoraria o engajamento das mulheres.

A Aline falou de mais moderadoras mulheres (desculpe caso já tenha alguma,
não sei quem é ainda... Hehehe), além disso há mais coisas no TC (ou outro
ambiente) que vocês gostam e que os moderadores poderiam tentar implementar
aqui?

Tirando o óbvio que é "não ser babaca"... Hehehe...  há algo mais que não
moderadores (como eu) podem tentar fazer pra deixa-las mais confortáveis?

Em qui, 22 de mar de 2018 22:11, Yone <notifications@github.com> escreveu:

**Comment by diessica** [comment]
poxa, que complicado os relatos! sempre me senti confortável e segura por aqui! só não contribuo menos porque não visito consistentemente, geralmente espero pingar (?) algo no meu feed

**Comment by ktquez** [comment]
Confiança mulheres.

2 problemas percebi nos relatos:

- O primeiro é que tem muito babaca mesmo que desmerece qualquer pensamento de uma mulher, fato triste!
 
- Mas o segundo problema é contornável, pelo que me parece é falta de confiança (por conta do primeiro problema), podem ficar sabendo que também tem muito homem que tem medo de por algum pensamento/ideia, pois sempre tem os narcisistas tecnológicos, acham que detém a melhor solução/resposta, em um mundo onde existem dezenas de soluções para um problema.

Já no sentido contrário, percebo que quando há uma postagem de uma mulher, jorra comentários para ajudar (parecem pavões).

Mete a cara e faz, sei que é difícil, mas com certeza o desafio vale a pena pela recompensa da conquista.

Eu fico extasiado com Lea Verou, Sarah Drasner, Sara Soueidan, Jessica Lord e Loiane Groner 
(A concentração, foco e comprometimento no conteúdo passado pela mulheres é extraordinário).

**Comment by n3pu** [comment]
Não se sintam acuadas, eu também ja me senti acuado. Defenda o que você acha ser certo e aprenda quando estiver errada(o). Sempre vão ter as pessoas escrotas, mas vai por mim, se sentir desconfortável e acuada é pior. Bola pra frente, ninguém sabe de tudo e ao mesmo tempo você sabe alguma coisa que possa contribuir. Cada dia é um aprendizado diferente. E se tiver duvidas que não tenha issue aberta, vai lá e abre uma. 
Suas dúvidas podem ser de outras pessoas assim como eu ;).

**Comment by lucasjs** [comment]
Em tantos anos de mulheres lutando, participando, subindo em palcos, ajudando comunidades, não tem UMA moderadora aqui? Queria só saber o motivo. Ninguém nunca pensou nisso desde o começo até hoje? Em tanto tempo?

**Comment by brunoravanhani** [comment]
Eu concordo um pouco, com o @ktquez porque essa questão e confiança é algo que falo pra mim mesmo, que mesmo sendo homem, muitas vezes me falta confiança para responder ou perguntar algumas coisas.

É claro que é muito escroto o cara que responde mal para a pessoa, seja por ser mulher ou iniciante, mas nesse grupo, na minha opinião, os moderadores são muito presentes e sempre fizeram um bom trabalho para evitar esse tipo de coisa, ou excluir, caso aconteça.

Enfim, sintam-se bem-vindas, acredito que a maioria aqui gostaria de ter mais presença feminina aqui.

**Comment by victorsilent** [comment]
Falando por mim que tenho algumas questões que as meninas acima citaram, aqui no frontendbr foi um dos únicos locais de discussão que até hoje não presenciei alguém sendo desrespeitoso com o outro, talvez por ser um ambiente onde o seu perfil de fato é exposto (muitos usam o github como curriculo/portfólio) as pessoas escrotas ficam na sua :v
E outro ponto é que ver as pessoas sempre preocupadas em tecer um bom comentário para de fato auxiliar na dúvida existente sempre me fez ter confiança de que se alguém for escroto essas pessoas iriam o repreender <3

**Comment by bernardodiasc** [comment]
> Por que as minas não participam do fórum? 

Tem várias mulheres que participam do forum! 💃

> Vocês, mulheres, ficam desconfortáveis, são tratadas mal ou algo do tipo aqui no fórum?

... 👂👂

**Comment by hmilena** [comment]
A insegurança vem do problema de ser tratada diferentemente ou como inferior. O meio de tecnologia / gamer / geek acaba que é basicamente uma panelinha. Já perguntaram, por exemplo, pras meninas que jogam videogame, o que elas escutam quando jogam online? É complicado. Acho que o receio é o mesmo pra tecnologia.

Também há um processo nisso. Desde muito pequena eu jogo, gosto de computadores e via claramente as pessoas me olhando de forma diferente, falando coisas, apontando, etc, porque existe essa construção social que diz que videogame, computador, tecnologia, etc, pertence ao universo masculino, mesmo tendo um monte de mulheres que sempre estiveram envolvidas com grande parte disso tudo.

Tá mudando, mas é um processo que ainda vai demorar.

**Comment by willycamargo** [comment]
No início desse ano eu fiz a leitura de um livro chamado **Os homens explicam tudo para mim**, da  escritora **Rebecca Solnit**.

Desde o título o livro me ajudou a mudar a forma com que vejo o diálogo entre mulheres e homens. Confesso que, por mais que eu me esforçasse, eu não entendia e tinha alguns dos preconceitos que o livro aponta.

Acontece que, historicamente, a sociedade foi educada com mulheres ouvindo mais e dando menos opiniões. Isso está nas ruas, nas escolas, nas religiões, nas empresas, na política, dentro das casas etc.

É comum, entre vários grupos, os homens serem vistos como "portadores da verdade". O livro mostra o quanto isso inibe a mulher, porque durante toda sua vida quando ela tenta expor sua opinião e outro homem expõe uma opinião contrária, normalmente as pessoas costumam aceitar a opinião do homem. Esse mesmo fato colabora para que os homens se sintam mais confiantes com o tempo.

Para quem tem olhos e quer enxergar, basta dar uma olhada em comentários do facebook, twitter, linkedin, youtube... Também conseguimos encontrar isso com mais força em jogos online e, pasmem, em **fóruns**.

Nós, homens, normalmente não temos medos de dar uma opinião, mesmo quando ela pode estar errada. Já as mulheres precisam ter certeza daquilo que estão falando, porque se um homem contrariar ela e ela não conseguir provar o ponto de vista dela, provavelmente irão acreditar nele (mesmo que ele não prove o ponto de vista dele também).

Claro que nesse contexto existem exceções, mas infelizmente o que disse acima costuma ser o "padrão".

Minha dica vai principalmente para a maioria dos homens: parem de ser babacas e identifiquem que vocês possuem esse preconceito. Passem a interpretar as opiniões de mulheres com a mesma relevância com que interpreta as opiniões de homens e, sempre que possível, repreenda o amiguinho que menospreza as opiniões de mulheres.

**Comment by alinebastos** [comment]
Lendo o comentário acima lembrei de uma coisa que acontece muito, mas passa despercebido: Alguém faz uma pergunta, uma mulher responde, um homem responde praticamente a mesma coisa, os agradecimentos vão pro homem.
Vou citar o TC novamente, mas é um lugar, talvez o primeiro, onde eu respondo, e a pessoa que responde em seguida me cita "acrescentando ao que a Aline disse", ou algo do tipo, não passando por cima simplesmente do que eu disse, como se eu não tivesse dito nada, como acontece na maioria dos lugares.
Lógico que não são todos lá que fazem isso, mas a maioria das pessoas lá está notando esse comportamento e seguindo.

**Comment by alinebastos** [comment]
Então não é questão de ter confiança, de erguer a cabeça, são essas pequenas coisas que vão nos desestimulando. E, como já citaram ali em cima, não é de um dia pro outro, requer trabalho, principalmente dos homens e dos moderadores.

**Comment by liviagabos** [comment]
É um trabalho de educação gradual.
Se virem alguém fazendo isso, dar uma cutucada e falar "pessoa, isso não está certo...olha a pessoa x falando tal coisa"
Acho que não apenas ter uma moderadora, mas ter alguém preocupado com a inclusão e ficar de olho onde ela não existe. No TC existem N pessoas de todos os gêneros preocupadas com a inclusão. Quando acontece de ter alguém contestando algo da luta do feminismo (por exemplo), não precisamos esperar uma mulher ir lá e falar com a pessoa. Qualquer um do TC vai falar com a pessoa e explicar que aquilo não está certo. 
É um trabalho de paciência e tolerância enorme, mas que infelizmente (ou felizmente pq temos espaço para isso) tem que ser feito.

E principalmente, esse trabalho de educação deve ser endossado por pessoas de grande impacto da comunidade e/ou moderadores.  Para que as pessoas entendam e aprendam que aqui (como em qquer outro lugar) a falta de inclusão não será tolerada.

**Comment by ktquez** [comment]
@alinebastos 
Lhe desestimula desejar que vocês que tenham mais confiança? A auto-estima é fortificante.

Quando eu citei antes no comentário acima, não foi a intenção dizer que só falta isso para tudo se resolver, pelo contrário o progresso começa por ai.

---

Eu adoro as pessoas (Homens e mulheres) que pensam: 
`"Foda-se quem achar ruim ou ser um infeliz, eu vou falar o que penso, talvez eu ensine, talvez eu aprenda, mas não me calo."`

**Comment by alinebastos** [comment]
É que é fácil pra um homem falar pra gente ter confiança, quando na verdade não é esse o problema :)

**Comment by luizbills** [comment]
Nos comentários acima disseram sobre o povo bacaca ser minoria, mas o problema que muitas vezes essa minoria tem muita voz. Alguém chega com linguagem agressiva, é retrucado com mais linguagem agressiva e a moderação muitas vezes não faz nada, deixando elas por elas. O caso acaba se repetindo várias vezes com outros participantes ou até mesmo com os anteriores. Sempre acaba sendo visto como uma "discussãozinha de comentários".

Também já estou cansado de ver no facebook (aqui já vi também) alguém perguntando algo ou até mesmo respondendo, então vem um colega faz um comentário idiota (que as vezes até pensam ser inocente) e vários acham engraçado, colocam emoji e até completam a piada. Novamente nada é feito.

E sempre var ter aquela desculpa: moderadores são pessoas ocupadas. Mas no final quase não tem moderação mesmo. 

As poucas vezes que eu fui moderador, eu era aquele chato que importava com as regras (sempre lembrando delas) e chamando no privado os trolls para tentar dar uma chance ao coitado antes de banir ele.

Tenho certeza que ninguém esquece que tem pessoas do outro lado, mas muitos não pensam duas vezes antes de se querer ser maior que os outros e estragar o dia de muitos.

Então seja homem ou mulher, sugiro pensar assim: eu posso ajudar? ótimo, vou responder. Não posso ajudar? vou marcar alguém que possa, vou apenas observar ou ignorar mesmo. Evita fazer comentário desnecessário. Evita agredir um agressor também, prefira marcar alguém da moderação. É muito fácil ser um babaca, então faça uma força muito grande para não ser.

**Comment by luizbills** [comment]
@ktquez 
> Lhe desestimula desejar que vocês que tenham mais confiança? A auto-estima é fortificante.

Cada caso é um caso. Isso de ficar dizendo: "é só ter coragem, vai lá que você consegue"; é a mesma coisa que dizer: "se ele consegue, eu também consigo". Qualquer pessoa que fizer um minimo de esforço, vai descobrir que não é assim que funciona.

**Comment by ktquez** [comment]
@luizbills 
> ..., não foi a intenção dizer que só falta isso para tudo se resolver, pelo contrário o progresso começa por ai

Você falou o que eu tinha afirmado antes.

Nem tudo na vida se resolve com uma única coisa, mas essa única coisa pode ser o primeiro passo e esse passo pode ser o mais importante.

**Comment by luizbills** [comment]
@ktquez eu gosto de acreditar que o primeiro passo tem que ser feito pela pessoa ou a pessoa tem que estar pelo menos preparada para esse primeiro passo. É muito fácil chegar para alguém e achar que temos uma solução para o problema dela. Novamente, cada caso é um caso, **tudo requer muita comunicação e reflexão**. Então é natural, ser algo complicado e de dificil compreensão ou aceitamento. Só não podemos é deixar isso enfraquecer nossa vontade ajudar ou achar que é *mimimi* da outra parte.

**Comment by alinebastos** [comment]
Deixando claro aqui que o primeiro passo não é a mulher ter mais confiança, levantar a cabeça e tal. Não se sentir confortávem aqui neste fórum não quer dizer automaticamente que não temos confiança fora daqui.
Quem falou isso está simplesmente supondo que não temos confiança ou auto-estima, ou seja, a culpa é nossa.
O primeiro passo é do forum, dos integrantes e moderadores do forum, inclusive foi por isso que abriram essa issue, eles querem saber o que ELES podem fazer.

**Comment by lucasjs** [comment]
Rapazes parem com essa de "isso tbm acontece comigo", "comigo homem tbm é assim", etc. Não, vc não sabe como é. Deixem as mulheres falarem.

**Comment by bernardodiasc** [comment]
Acho curioso os caras aqui respondendo as perguntas que foram explicitamente direcionadas para as mulheres... Não seria exatamente esse um ato de silenciar a opinião feminina? Tem muita mulher que participa do forum sim, deixem-nas responder o tópico oras... :P

**Comment by alinebastos** [comment]
> Não seria exatamente esse um ato de silenciar a opinião feminina? Tem muita mulher que participa do forum sim, deixem-as responder o tópico oras

EXATAMENTE!!!!

**Comment by willycamargo** [comment]
> Acho curioso os caras aqui respondendo as perguntas que foram explicitamente direcionadas para as mulheres... Não seria exatamente esse um ato de silenciar a opinião feminina? Tem muita mulher que participa do forum sim, deixem-nas responder o tópico oras... :P

Você tem razão, @bernardodiasc . E por mais contraditório que possa parecer, é sobre isso que tentei falar no meu comentário: sobre como interpretamos e nos colocamos a frente delas sem percebermos.

Não irei apagar meus comentários, mas peço desculpas por ter respondido mesmo sendo direcionado diretamente para mulheres. E a única coisa que posso me comprometer é em ficar mais atento a esses pequenos detalhes a cada dia.

A thread é de vocês, **Mulheres**! 👩 

Diga-nos o que podemos fazer para vocês se sentirem mais confortáveis, esse espaço **deve ser de todos**.

**Comment by n3pu** [comment]
A intensão é justamente dar voz. Compreendam que ler relatos é essencial para o entendimento de ambas as partes. O ser humano é complexo, ou então não precisaria disso tudo não é? 
Querem complicar mais ainda dizendo tudo isso sobre culpar ou não alguém, apontar o dedo se desviando do problema não é resolver.
Isso é uma comunidade, uma sociedade, que tem que tratar problemas com soluções e para soluções, precisamos de discursão.
Dar voz a **elas**, no sentido de abrir a issue é justamente entender o fato.

Concluindo e concordando que o espaço deve ser de **todos**.

**Comment by eduardojmatos** [comment]
Oi gente!

Nós ainda não temos mulheres na organização, de fato. Mas é pelo motivo de ainda não termos parado um tempo pra fazer essa análise de quem poderia integrar a org, que seja engajada com o projeto como um todo. Já temos nomes, mas de verdade não paramos pra definir isso, somente discutimos (lá pros meados de 2016). 

Logo vamos adicionar mais pessoas a outros projetos, mas entendam que esse projeto, apesar de ser da comunidade, está sob os cuidados de pessoas que tem tarefas além da org, como trabalho, família, etc.. Queríamos fazer várias outras coisas, mas geralmente só quem já está envolvido(a) com muita coisa, acaba fazendo mais ainda. E não queremos adicionar pessoas só por adicionar, já que isso só atrapalharia quem gostaria e tem tempo de fazer de verdade.

Se tiverem sugestões de meninas que realmente são ativas por aqui, podem mandar pra gente. Já temos nomes, mas seria legal validar com vocês, quem seria interessante termos na org.

Muito bacana a discussão, acho bem válido levantarmos esses pontos.

**Comment by vitor-mendrone-deel** [comment]
Fala pessoal, tudo bem? 

Apenas complementando alguns pontos da mensagem do @eduardojmatos: 

Hoje o fórum tem um grupo de admins que são os mesmos que iniciaram o projeto. Esses admins (@LFeh, @mtmr0x, @willianjusten, @fdaciuk, @vitorbritto, @eduardojmatos, @diogomoretti e eu) não atuam como moderadores a não ser que se faça necessário, ou seja, que o código de conduta seja infringido. Nosso papel é basicamente o de manter a organização, remover vagas fora do padrão e divulgar o fórum, ou seja, não existe nenhum papel de moderador como nos antigos fóruns

O fórum não tem, salvo o tema, nenhuma ligação com o grupo Frontend Brasil do Facebook que é administrado por um outro grupo de pessoas. Esse forum foi criado justamente para evitar os problemas que existiam lá. Convenhamos, aquele grupo parece a pãgina de comentários do G1. 

Como disse o Edu, estudamos a algum tempo a inclusão mulheres, porém nosso principal critério seria o de adicionar uma mulher que ja fosse ativa e participativa na org como um todo. 

Por fim é importante lembrar que o FrontendBR e o Training Center existem com um objetivo em comum (ajudar o desenvolvimento de profissionais e da comunidade como um todo) mas nascem de ideias completamente diferentes. 

Nosso trabalho aqui ja ajudou dezenas de pessoas a conseguirem novos empregos, resolverem problemas e conhecerem novos eventos. Também deu vida a diversas iniciativas, o proprio Training center por exemplo e a diversos outros fóruns e comunidades. 

Contamos com vocês para continuarmos com esse trabalho e lembro a todos que essa é uma iniciativa open source.

**Comment by akfzambrana** [comment]
Vem aqui a minha opinião...

Mulheres são julgadas por todo mundo, tanto por homens como por mulheres. Elas são julgadas por ser feias, bonitas, atleticas, altas, magras, gordas, solteiras, casadas, mães, homosexuais, bisexuais, e qualquer outro motivo que possa ser diferente.

Este é um projeto opensource hospedado em um lugar publico, que qualquer pessoa, seja bem intencionada ou não, pode ler, então eu acredito que muitas mulheres não participem e não deem suas opiniões por medo de serem mais julgadas ainda. Especialmente por empresas e pessoas que fazem recrutamento e isso possa prejudicar mais ainda a sua carreira.

Existem temas complicados e polemicos que tendem a ser julgados de forma bem extremista por algumas pessoas, logo isso poderia acabar inibindo mulheres que se preocupam com suas carreiras e seus relacionamentos com as empresas em que trabalham.

Novamente, esta é uma opinião, por ser opinião, é pessoal e não um argumento de que esse é o motivo, mas pensando cá com meus botões e conhecendo as historias de muitas mulheres fodas que tem suas carreiras prejudicadas por "aparecerem mais" quando se trata de assuntos "polêmicos" chego a essa conclusão.

**Comment by miguelzinhe** [comment]
Acho que, com essa pergunta, veio também exemplos ótimos do que estamos falando em questão dos homens quererem saber e tentar entender/ajudar negativamente uma mulher quando não solicitado, entende? A thread era para MULHERES responderem, guyssss!!!! Daí chegou uns caras falando "mulheres, mais confiança, avante!". Cara, hmmm, isso não é uma questão de confiança ou necessita de uma validação (ou não validação) masculina pra ser real. É a realidade. 

Nós saímos sempre prejudicadas quando vamos nos expor fora das nossas bolhas de amigos dentro de TI. E isso não é não ter confiança, e sim tentar sofrer menos com o preconceito velado e estruturado dentro, principalmente, da ciência e da tecnologia. Por exemplo, antes de palestrar nos lugares, acho que a maioria das minorias aqui pesquisa o histórico da comunidade e do lugar pra não acontecer coisas terríveis que afetem nosso psicológico. Pra que não aconteça da gente desistir do nosso sonho e da nossa carreira pela discriminação, sabe?

Talvez não tenha sido a intenção e talvez tenha sido algo total no impulso que alguns caras responderam isso aqui "tentando incentivar". Mas, percebe que esse impulso masculino de sempre validar uma mulher quando ela relata coisas é enraizado na nossa sociedade? 

Eu não quero tretar, sabe? Eu também só expus minha opinião e não precisava um grande esclarecimento ou uma moderadora mulher. Eu sei que a gente trampa. Somos profissionais liberal ou proletariados e temos nossas coisas, mas, o mais importante é reforçar sempre (e também com atitudes) que existe um código de conduta e que vocês são aliados das minorias sociais, sabe? Que tá tudo bem nós nos expormos porque, não basta só dar kick ou block no sujeito, é nos defender. É defender pessoalmente mesmo.

Eu apenas disse que tenho medo de dar minha opinião porque eu sinto que vou ser julgada ou silenciada sempre, como a @akfzambrana disse. Que meu conhecimento não vai servir de nada quando um cara vai vim e falar a mesma coisa. Além disso, reforçando tbm o que a Andrea disse, as empresas estão de olho. A comunidade está de olho. E isso pode ser prejudicial. Só de eu estar escrevendo esse textão pode ser prejudicial pros meus processos seletivos. 

Esse é o mal de ser uma desenvolvedora hoje na área, entende? Tudo que eu, pessoalmente, queria é que meus colegas homens me defendessem e eu pudesse contar com eles.

**Comment by priscilacng** [comment]
Falando especificamente de mim, não consigo participar do fórum pq sempre tem muita coisa rolando ao mesmo tempo e quando eu vejo já falaram o que eu gostaria de ter falado hahahahaha

Por isso em geral eu me manifesto mais no canal do slack, que é mais fácil pra mim :)

Mesmo assim, isso não tira o fato de que já trabalhei com pessoas que se surpreendiam que eu era boa, afinal eu era mulher. É engraçado e ao mesmo tempo triste como as mulheres são subestimadas

Como o @lucasjs falou, os homens não sabem como é que as mulheres se sentem _(na vida, não só na área profissional)_ e muitas vezes nos menosprezam, como se o preconceito fosse algo que não acontece mais

O machismo está enraizado em todo mundo amiguinhos, em alguns menos do que em outros.. é uma coisa complexa, mas tudo se resolveria se todos praticassem a **empatia**

**Comment by jezzipan** [comment]
**Meus pontos de vista e consideração.**
Sou nova na comunidade, uso muito o fórum para entender e saber o que está acontecendo de útil na comunidade.

O que vejo próxima na minha realidade. Já participei de iniciativas de mais mulheres na tecnologia em especial no front (PrograMaria, Reprograma...) 

Essa pergunta é muito boa e com ela vou fazer de tudo para vocês, homens, tenham mais dúvidas. 
Será que existe muitas mulheres  que conhecem e usam Git e github? Na vida, quando encontro uma colega de frontendbr que conheci em algum meetup, essas iniciativas e afins elas me falam sobre a correria que é pra conseguir um freela básica (HTML, CSS, JS) eu recomendo a comunidade frontendbr-vagas no github e muitas não conhecem, mas conhecem o grupo hostil do Facebook. 

E aí, como incentivar as mulheres a usar essa plataforma? 


Eu tenho um padrinho na área, o @RatoX  e sinceramente, sem ele não seria nada e reconheço muito o meu privilegio em conhecer pessoas como ele. Eu sei que eu, enquanto mulher, negra, lésbica (sim meus caros colegas, tenho que pontuar porque o mercado cobra e caro ainda) o quanto seria difícil meus espaço aqui. Minha ideia não é dar biscoito pra ninguém mas reconhecer pessoas que me salvaram. 

Apesar de nunca comentar aqui por questões de correria (estudo, trabalho, família...) eu me sinto ok neste ambiente, percebo que existe profissionais com o mínimo de educação mas percebo que também estou muito atrasada em questões básicas.


Esse fórum é mais organizado, profissional e técnico do que aquela porcaria que tem no Facebook. 

Percebo que muitas iniciativas de colocar mais mulheres no Front-End se limitam muito ao básico e isso não é um problema. Nós mulheres estamos num corre cabuloso para tentar entrar no mercado mas tb não acompanhamos um assuntos "avançados" tipo pwa saindo do HTML, CSS e JavaScript básico.

Existe mulheres maravilhosas na área como Lorraine, Vanessa Tonini, Geysi Domiciano, Talita Pagani, Alda Rocha...? Existe! Mas ainda são poucas e pontos "fora da curva" e que mesmo assim fazem um trabalho de formiguinha para acabar com a violência na comunidade. 

Outro ponto, colocar mais mulheres moderadoras mas quais serão as funções delas? Apenas ensinar como o colega de vcs estão sendo machistas? Eu quero ver mulher sentando o dedo no react, travando o nosso cérebro de informação. Homens lidem entre vcs com suas atitudes tóxicas, uma coisa é pedir ajudar mas reconheçam tb entre vcs.  

A pergunta também deveria ser: como suprir o gap feminino nas empresas e por que as mulheres estão falando tanto sobre direitos básicos e ensinando os caras a não seres escrotos enquanto o certo era nós, enquanto comunidade, estarmos todos engajados no uso de TDD, pwa e afins? 

**Quer ajudar?** 

Aprenda a ouvir/ler outros pontos de vista. O mundo não é só meme e lol e mitou.
Tente e ajude a limpar o ambiente. Viu um post agressivo na comunidade? Se manifeste contra. Utilize seu poder de fala.

Abra um medium / canal no youtube,  telegram... ensine e engaje coisas 'básicas' e espalhe este conteúdo. Tente se aproximar de iniciativas com objetivo de dar mais acesso às mulheres. 

Vá a um cursinho pré-vestibular popular próximo de vc e ensine física, química, matemática

Ofereça para ser padrinho de alguma mulher. 

O que estou comentando aqui não tem NENHUMA relação com" proselitismo ideológico" é só vcs verem o gap que as empresas de TI no Brasil tem, os dados demográficos do Stack over flow e comparar com a realidade da nossa comunidade brasileira. 
Abc.

**Comment by RatoX** [comment]
_Homens lidem entre vcs com suas atitudes tóxicas, uma coisa é pedir ajudar mas reconheçam tb entre vcs._

Aprendi um monte nessa thread mas isso me marcou. Obrigado @jezzipan

**Comment by AmandaFonseca** [comment]
Essa issue engloba vários assuntos que estão me "atormentando"  atualmente ...

Primeiramente esse post é para as meninas acredito na boa intenção de alguns que responderam mas a pergunta é "Por que as minas não participam do fórum? "

Eu estou começando na área e já ouvi um cara do meu serviço falar: "Mulher não sabe o que é commit" nesse momento eu ainda estava trabalhando com SAC me virei para ele e disse : Eu sei kkk

Na hora ele fechou a cara e começou a me testar perguntando o que era um fork? o que é uma branch? 
E soltou um: Acho que não tem trabalho no SAC hoje....

 Ninguém da panelinha chama a menina que esta a fim de ser front para sentar na mesa e compartilhar alguma coisa nova com ela porque se ela sentar na mesa todo mundo vai ter que parar de falar "besteiras" ( eles é que pensam que são discretos e que ninguém escuta o que eles falam).

São essas e outras atitudes que acabam com a confiança de muitas meninas que chegam com toda a vontade de mostrar o valor de seu trabalho.  Como a @jezzipan bem falou todas as iniciativas voltadas para atrair mulheres para a área são muito básicas e fora da curva fora do que o mercado realmente pede.

Já percebi que para uma menina realmente trabalhar como Dev ela tem que ter muitos mais requisitos do que a vaga pede ela tem que ter saber o que ninguém da empresa sabe, porque se ela depender somente de feedback da equipe pode ser dispensada.

Por enquanto sou uma mera aprendiz ainda procurando construir uma boa base de skills mas espero sim um dia compartilhar meus conhecimentos com outras meninas.

Sobre esse fórum especificamente eu não tenho nada a reclamar sempre acompanho e comento sem problemas porém, como a @oieusouamiguel  mencionou a comunidade do Facebook é cheia de cara escroto.

**Comment by danielschmitz** [comment]
Este é o post com mais coraçõeszinhos do forum 😁

**Comment by cyberglot** [comment]
Vou dar minha opinião como quem já esteve dos 2 lados da conversa: quem pergunta e quem responde. E falo sobre fóruns de programação e tecnologia em geral, e não desse em específico porque eu participei poucas vezes para saber qual é o clima.

**Quem pergunta**: quando você é mulher (ou notoriamente de outra minoria, por exemplo, angolano), as pessoas que vão te responder assumem coisas sobre você, em geral, que você não tem noção do que você está fazendo e te responde coisas obvias, muitas vezes sem ler o que você escreve. outras pessoas se irritam por ver que você, mulher em tecnologia, pode estar trabalhando com isso, e te manda estudar mais, ridiculariza tua dúvida, te xinga. outras pessoas te tratam bem na esperança de te chamar no chat depois e perguntar se você tem namorado ou se quer casar. muitas mulheres fazem contas em fóruns com nome/avatar masculino pra evitar esse tipo de atenção.

**Quem responde**: quando você é mulher (ou notoriamente de outra minoria, por exemplo, angolano), as pessoas novamente assumem coisas sobre você, em geral, que você não tem noção do que você tá falando. irão te cobrar que você use norma culta da língua, sem erros ou _typos_, e irão te ridicularizar caso isso aconteça. também irão te cobrar que cada termo técnico que você use esteja de acordo com uma definição de um lugar específico que ele acha que é a mais certa (independente de ser ou não), e irá te humilhar por não saber "o básico" (independente de ser ou não). continuarão argumentando contra sua resposta certa até chegar um outro homem e confirmar que você está certa, e irão pedir desculpas pra esse homem e dizer que se confundiu ou quis dizer outra coisa. outras pessoas irão ignorar sua resposta certa até que um homem venha e responda exatamente a mesma coisa, e dai irão agradecer à ele. 

Por essas e outras eu que desengajei de 100% dos fóruns onlines e só faço perguntas no SO com uma conta com nome masculino.

**Comment by ghost** [comment]
Eu normalmente não participo porque ainda estou estudando. Eu só passei por uma ou outra situação de cara me interromper e achar que eu nunca tinha tido interesse em tecnologia. Pelo menos aqui o que me deixa meio de lado é minha inexperiência mesmo
